content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -718,3 +718,93 @@ Clean up the Pod when you're finished with it:
 ```shell
 kubectl delete pod myapp
 ```
+
+### Custom Profile {#custom-profile}
+
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
+
+You can define a partial container spec as a custom profile in either YAML or JSON format, and apply it to an ephemeral container or a debugging container in a copied Pod or a debugging Pod using the `--custom` flag.
+
+{{< note >}}
+Custom profile only supports the modification of the debugging container spec. It does not support the modification of the Pod spec of the debug target.
+Modifications via a custom profile are not allowed for certain fields, including: `command`, `image`, `lifecycle`, `name`, and fields that define access to storage.
+{{< /note >}}
+
+
+First, create a Pod named myapp as an example:
+
+```shell
+kubectl run myapp --image=busybox:1.28 --restart=Never -- sleep 1d
+```
+
+Create a custom profile in YAML or JSON format.
+Here, create a YAML format file named `custom-profile.yaml`:
+
+{{< tabs name="custom_profiles" >}}
+{{< tab name="YAML" codelang="yaml" >}}
```

Comment:
Since the YAML and JSON files are equivalent, we don't need to show both.
The maintenance burden overweighs the convenience brought to users.

Reply:
I've fixed to show only YAML format file.